packages/frontapp/src/utils/managerCardTypes.ts

```diff
@@ -0,0 +1,5 @@
+export enum ManagerCardTypes {
+  OVERDUE = 'Overdue',
+  DONATES = 'Donates',
+  NOTIFICATIONS = 'New_user_notification',
+}
```

Comment:
This is not 'utilitary' data type, it's related to the domain/model, so it should be defined in appropriate place (usually near the component it used by).
Also, it's not necessary to define descriptions for enum values, just use keys.